### ISSUE TITLE
trackEvent eventDefinition not showing the correct parameters

### ISSUE THREAD

**Issue by pohzipohzi** [rangle/redux-beacon#307]
This is a bug in the documentation
----
 - [x] :beetle: Bug Report
 - [ ] :rocket: Feature Request
 - [ ] :scroll: Documentation Request

Which version of Redux Beacon are you using?
----
"redux-beacon": "^2.0.3",
"@redux-beacon/google-analytics-gtag": "^1.0.2",

Which target(s) are you using?
----
 - [ ] Google Analytics
 - [x] Google Analytics (gtag)
 - [ ] React Native Google Analytics
 - [ ] Google Tag Manager
 - [ ] React Native Google Tag Manager
 - [ ] Amplitude
 - [ ] Segment
 - [ ] Other/Third Party: ...(please specify here)

:beetle:  What did you expect to happen?
----
`EventDefinition` in `trackEvent` should be a number, not string. This comes from the documentation [here](https://support.google.com/analytics/answer/1033068)

:beetle: What happened instead?
----
Documentation shows this instead:

```
trackEvent: (eventDef: EventDefinition<{category: string, action: string, label?: string, value?: string}>) => EventDefinition<any>
```

Can you help out?
----
 - [x] :star2: I am a legend and can get started on a pull request right away given the go-ahead.
 - [] :star: I am a superstar and would like to help out given some guidance.
 - [ ] :disappointed: I won't be able to help out on this one.

**Comment by ttmarek** [comment]
Correct! Good catch. This is fixed and available in `@redux-beacon/google-analytics-gtag@1.0.4`
